RELEASING.md

```diff
@@ -3,23 +3,27 @@
 1. checkout master
 2. pull from repo
 3. run the unittests
-4. run `loghub` and update the `CHANGELOG.md` file:
+4. run `loghub`.  Replace <github username> and <previous version> with proper
+   values.  To get the previous version run `git --tag` and select the most
```

Comment:
The command is `git tag`, `git --tag` fails.

Reply:
Meh. First used `--tag` on command line and when writing the command to instructions had already forgot the correct form.